### ISSUE TITLE
torchvision version inconsistency occurs when installing StableDiffusionWebUI on MacOS

### ISSUE THREAD

**Issue by RikitoNoto** [LykosAI/StabilityMatrix#984]
Thank you for the great application. I'm enjoying using it! :smile:

### Package

Stable Diffusion WebUI

### When did the issue occur?

Installing the Package

### What GPU / hardware type are you using?

Apple M2 MAX

### What happened?

Package Modification Failed occurs when installing StableDiffusionWebUI on MacOS with below message.
This message indicated that the torchvision versions were different.
Therefore, when I set ```torchvision==0.18.1``` in ```Pip Overrides```, the installation was successful.

I think that the cause is maybe [this code](https://github.com/LykosAI/StabilityMatrix/blob/1ff6860b4bbdc0f342104a60023a118d40f33d32/StabilityMatrix.Core/Models/Packages/A3WebUI.cs#L220).(I'm sorry if I'm wrong)
```cs
TorchIndex.Mps
    => new PipInstallArgs()
        .WithTorch("==2.3.1")
        .WithTorchVision("==2.3.1")
```
It seems like ```.WithTorchVision("==2.3.1")``` is probably a mistake for ```.WithTorchVision("==0.18.1")```
https://pytorch.org/get-started/previous-versions/#v231

Sorry, I haven't read the code in detail so I haven't made a pull request

### Console output

```
Unpacking resources
Unpacking resources
Download Complete
Requirement already satisfied: pip in ./venv/lib/python3.10/site-packages (24.3.1)
Requirement already satisfied: wheel in ./venv/lib/python3.10/site-packages (0.44.0)
Collecting torch==2.3.1
  Using cached torch-2.3.1-cp310-none-macosx_11_0_arm64.whl.metadata (26 kB)
ERROR: Ignored the following yanked versions: 0.1.6, 0.1.7, 0.1.8, 0.1.9, 0.2.0, 0.2.1, 0.2.2, 0.2.2.post2, 0.2.2.post3, 0.15.0
ERROR: Could not find a version that satisfies the requirement torchvision==2.3.1 (from versions: 0.12.0, 0.13.0, 0.13.1, 0.14.0, 0.14.1, 0.15.1, 0.15.2, 0.16.0, 0.16.1, 0.16.2, 0.17.0, 0.17.1, 0.17.2, 0.18.0, 0.18.1, 0.19.0, 0.19.1, 0.20.0, 0.20.1)
ERROR: No matching distribution found for torchvision==2.3.1
Could not install stable-diffusion-webui (StabilityMatrix.Core.Exceptions.ProcessException: pip install failed with code 1: 'Collecting torch==2.3.1\n  Using cached torch-2.3.1-cp310-none-macosx_11_0_arm64.whl.metadata (26 kB)\nERROR: Ignored the following yanked versions: 0.1.6, 0.1.7, 0.1.8, 0.1.9, 0.2.0, 0.2.1, 0.2.2, 0.2.2.post2, 0.2.2.post3, 0.15.0\nERROR: Could not find a version that satisfies the requirement torchvision==2.3.1 (from versions: 0.12.0, 0.13.0, 0.13.1, 0.14.0, 0.14.1, 0.15.1, 0.15.2, 0.16.0, 0.16.1, 0.16.2, 0.17.0, 0.17.1, 0.17.2, 0.18.0, 0.18.1, 0.19.0, 0.19.1, 0.20.0, 0.20.1)\nERROR: No matching distribution found for torchvision==2.3.1\n'
   at StabilityMatrix.Core.Python.PyVenvRunner.PipInstall(ProcessArgs args, Action`1 outputDataReceived) in /Users/runner/work/StabilityMatrix/StabilityMatrix/StabilityMatrix.Core/Python/PyVenvRunner.cs:line 241
   at StabilityMatrix.Core.Models.Packages.A3WebUI.InstallPackage(String installLocation, InstalledPackage installedPackage, InstallPackageOptions options, IProgress`1 progress, Action`1 onConsoleOutput, CancellationToken cancellationToken) in /Users/runner/work/StabilityMatrix/StabilityMatrix/StabilityMatrix.Core/Models/Packages/A3WebUI.cs:line 255
   at StabilityMatrix.Core.Models.Packages.A3WebUI.InstallPackage(String installLocation, InstalledPackage installedPackage, InstallPackageOptions options, IProgress`1 progress, Action`1 onConsoleOutput, CancellationToken cancellationToken) in /Users/runner/work/StabilityMatrix/StabilityMatrix/StabilityMatrix.Core/Models/Packages/A3WebUI.cs:line 269
   at StabilityMatrix.Core.Models.PackageModification.InstallPackageStep.ExecuteAsync(IProgress`1 progress, CancellationToken cancellationToken) in /Users/runner/work/StabilityMatrix/StabilityMatrix/StabilityMatrix.Core/Models/PackageModification/InstallPackageStep.cs:line 19
   at StabilityMatrix.Core.Models.PackageModification.PackageModificationRunner.ExecuteSteps(IEnumerable`1 steps) in /Users/runner/work/StabilityMatrix/StabilityMatrix/StabilityMatrix.Core/Models/PackageModification/PackageModificationRunner.cs:line 35)

```

### Version

v1.10.0

### What Operating System are you using?

macOS

**Comment by mohnjiles** [comment]
This should be fixed as of the [latest release, v2.12.3](https://github.com/LykosAI/StabilityMatrix/releases/latest)

**Comment by RikitoNoto** [comment]
@mohnjiles 
Thank you for letting me know!
I will close this issue.
It was helpful.
